node_modules/oae-core/upload/js/upload.js

```diff
@@ -399,7 +399,6 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport', 'j
                 'url': '/api/content/create',
                 'dropZone': $('#upload-dropzone', $rootel),
                 'forceIframeTransport': useIframeTransport,
-                'replaceFileInput': false,
```

Comment:
@sathomas : What should we do with this in the other widgets it's being used in:

```
node_modules/oae-admin/importusers/js/importusers.js
node_modules/oae-core/uploadnewversion/js/uploadnewversion.js
```